src/main/java/by/custom/utilcalculator/domain/constants/CurrencyType.java

```diff
@@ -0,0 +1,15 @@
+package by.custom.utilcalculator.domain.constants;
+
+public enum CurrencyType {
```

Comment:
Where it's possibly better to avoid words that don't give additional sense.
```suggestion
public enum Currency {
```

Reply:
Done.